src/components/ButtonPanel.js

```diff
@@ -1,35 +1,36 @@
 import React from 'react';
+import Button from './Button';
 
 const ButtonPanel = () => (
   <div>
     <div className="group1">
-      <button type="button"> AC </button>
-      <button type="button"> +/-</button>
-      <button type="button">  % </button>
-      <button type="button">  ÷ </button>
+      <Button name="AC" color="rgb(240, 241, 243)" />
+      <Button name="+/-" color="rgb(240, 241, 243)" />
+      <Button name="%" color="rgb(240, 241, 243)" />
+      <Button name="÷" />
     </div>
     <div className="group1">
-      <button type="button"> 7 </button>
-      <button type="button"> 8 </button>
-      <button type="button"> 9 </button>
-      <button type="button"> X </button>
+      <Button name="7" color="rgb(240, 241, 243)" />
+      <Button name="8" color="rgb(240, 241, 243)" />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.